src/sentry/integrations/jira/views/issue_hook.py

```diff
@@ -119,7 +119,6 @@ def get(self, request: Request, issue_key, *args, **kwargs) -> Response:
                 external_issue = ExternalIssue.objects.get(
                     integration_id=integration.id, key=issue_key
                 )
-                # TODO: handle multiple
```

Comment:
I think this comment was referring to multiple `GroupLink` objects coming back.

Reply:
Ahh, in that case I'll make it a bit more clear

Reply:
^ yeah this is about how we currently only handle one linked issue in the Jira glance view.

Reply:
Updated!